bin/infrasim-init

```diff
@@ -54,20 +55,21 @@ def create_infrasim_directories():
         os.system("rm -rf /usr/local/libexec")
         os.mkdir("/usr/local/libexec")
 
+
```

Comment:
@fub2 User should be aware that infrasim-init shall be called only once at the very beginning, or when they have to recover to the start point. If they want to keep configuration changes, they may copy, git, ...
Give the decision to user.